back-end/src/lms/views/absences.py

```diff
@@ -62,6 +65,30 @@ class AbsencePermission(BasePermission):
     ]
 
 
+class AbsenceAttachmentPermission(BasePermission):
+    permission_class = "absence-attachment"
+    view_name_rus = "Приложения"
+    methods = ["get", "delete"]
+    scopes = [
+        Permission.Scope.ALL,
+        Permission.Scope.MILFACULTY,
+        Permission.Scope.MILGROUP,
+        Permission.Scope.SELF,
+    ]
+
+
+@extend_schema(tags=["absence-attachment"])
```

Comment:
Probably, with
```suggestion
@extend_schema(tags=["absences"])
```
it would be easier to find in Swagger.